### ISSUE TITLE
Storage Usage

### ISSUE THREAD

**Issue by jmeasey** [afawcett/declarative-lookup-rollup-summaries#394]
hi, 

we have several lookup rollup summaries running on schedules, some every day, some once a week.  They are working great, but i have noticed a potential issue.

Looking at my storage usage, Lookup Rollup Summary Schedule Items is currently using 611.00MB and growing.  With the way SF storage is, im keen to avoid something eating up this much space.

Is there a way to limit this, and delete old ones?  Is there a setting I've missed somewhere?

thanks in advance

james

**Comment by afawcett** [comment]
These should be getting cleaned up as they are processed by your the RollupJob apex class you have scheduled? What i'm wondering though is if you have used the other scheduled mode, the full  recalc scheduled via the Scheduled Calculate button? And still have the Calculation Mode set to Scheduled. This i admit is a bit confusing, but if your using full scheduled re-calc you can you use some other Calculation Mode like Process Builder, this will avoid the DLRS triggers recording changes that are never processed by the RollupJob in your case. If this is your case, you can use the Truncate Object feature to clear the Lookup Rollup Summary Schedule Items out. I'll give some thought on how to better differentiate the two schedule modes better.

**Comment by jmeasey** [comment]
hi, 

thanks for this.

So, if ive understood correctly....I have changed the Calculation Mode on each of the summaries to Process Builder.  (They were all set to scheduled before & i had scheduled them using the Schedule Calculate button)

I've also then deleted all Lookup Rollup Summary Schedule Items records from the object (did this via Jitterbit, as i couldn't see a truncate option for that object).

These summaries run weekly, so i will check them next week and make sure they are working ok.

thanks again.

james
